create-container.sh

```diff
@@ -202,32 +206,28 @@ echo "Container has IP address: $IP_CONTAINER"
 echo
 
 # Add container IP to /etc/hosts
+
 HOST_ENTRY_COMMENT="# LXC container for $NAME"
-HOST_ENTRY="$IP_CONTAINER        $HOST"
-
-echo "Removing old host $HOST from /etc/hosts"
-# When deleting it's critical to control the pattern, in this case,
-#+ with start (spaces) and end ('$' end of line), as a simple matching
-#+ against /$HOST/  could be harmful.
-#+ In this supposit, running devenv for odoo-1 would delete any entries
-#+ of odoo-12, and those of my-new-odoo-1.domain
-# Example:
-# NAME="Odoo 1"; HOST="odoo-1"
-# HOST_ENTRY_COMMENT="# LXC container for Odoo 1"
-# HOST_ENTRY="10.10.10.10        odoo-1"
-# echo "${HOST_ENTRY/$IP_CONTAINER/}" →  "        odoo-1"
-# See man bash(1) and search for 'Parameter Expansion'
-sudo sed -i "/${HOST_ENTRY/$IP_CONTAINER/}$/d" /etc/hosts
 sudo sed -i "/^$HOST_ENTRY_COMMENT$/d" /etc/hosts
-
-echo "Add entry '$HOST_ENTRY' with comments to /etc/hosts"
 sudo -- sh -c "echo \"$HOST_ENTRY_COMMENT\" >> /etc/hosts"
-sudo -- sh -c "echo \"$HOST_ENTRY\"         >> /etc/hosts"
-echo
 
-# Remove host SSH key
-echo "Removing old $HOST from ~/.ssh/know_hosts"
-ssh-keygen -R "$HOST"
+for HOST in $HOSTS; do
+  HOST_ENTRY="$IP_CONTAINER        $HOST"
+  echo "Removing old host $HOST from /etc/hosts"
+  sudo sed -i "/$HOST_ENTRY/d" /etc/hosts
+  sudo sed -i "/$IP_CONTAINER/d" /etc/hosts
```

Comment:
Hi! I think these 2 lines are dangerous. When the expressions are not bounded, they can match on similar, longer ones.

For instance, line 217 would delete both lines in this case:
```bash
# HOST=odoo.coopdevs.org
# IP_CONTAINER=192.168.122.10
192.168.122.10        odoo.coopdevs.org
192.168.122.10        odoo.coopdevs.org.local
```
It's an edge case, but I would add start and end line delimiters:
`sudo sed -i "/^$HOST_ENTRY$/d" /etc/hosts`

Line 218 I find it more dangerous, cause it's a smaller part of a line. This case is more possible:
```bash
# HOST=odoo.coopdevs.org
# IP_CONTAINER=192.168.122.10
192.168.122.100        odoo.processos.org
192.168.122.10        odoo.coopdevs.local
```
Both lines would be deleted because both contain the string `192.168.122.10`. In this case, I would try again to set the boundaries:
`sudo sed  -i "/^$IP_CONTAINER /d" /etc/hosts`
The left boundary, line start, and the right boundary a blankspace. You can use something more visual as \w or \\> for generic white or for word delimiter, but this one works universally and the other methods I'm not sure of the platform scope they actually work in.

Finally, domains that share the same IP can be written in the same line. Maybe it's easier to do `HOST_ENTRY=$IP_CONTAINER        $HOSTS`. To delete, it would work the same way, although it should be easy to find simpler ways.